test/src/runTest.js

```diff
@@ -34,6 +50,10 @@ async function recordBrowser(state, test, testPath, browserName) {
     await browser.close();
   }
 
+  if (updateWebsocketLogs) {
```

Comment:
it'd also be nice to have a field in `manifest.json` for specifying if we should save a fixture

Reply:
We should compare the new and old fixture and warn locally and fail in CI.

Reply:
> We should compare the new and old fixture and warn locally and fail in CI.

Originally I thought we'd just run tests and only complain if they failed but I like this suggestion better.